requirements.txt

```diff
@@ -1,6 +1,6 @@
 flaky
 numpy
-pennylane>=0.15
+git+https://github.com/PennyLaneAI/pennylane.git@master
```

Comment:
Will this be changed to `pennylane>=0.16` after the release?

Reply:
The idea would be to merge the release candidate into `master` after the release. It seems, that there we're still pinning to `pennylane>=0.15`:
https://github.com/PennyLaneAI/pennylane-lightning/blob/937592796828ad131469a9258e715c7c7994c724/requirements.txt#L3

Having said that, let me undo this line, the change was made just to make sure that tests pass well with the `master` branch of PennyLane.

Reply:
```suggestion
pennylane>=0.15
```